<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreDialog</class>
 <widget class="QDialog" name="ScoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scores</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="easyTab">
      <attribute name="title">
       <string>Easy</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="mediumTab">
      <attribute name="title">
       <string>Medium</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="hardTab">
      <attribute name="title">
       <string>Hard</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="insaneTab">
      <attribute name="title">
       <string>Insane</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
